spec/season_stats_spec.rb

```diff
@@ -12,7 +12,7 @@
     @games_teams_fixture_path = "./data/games_teams_fixture.csv"
     @locations =
       {
-        games: @games_fixture_path,
```

Comment:
We did all of our original testing on the games_fixture, but found it was difficult due to the need of both game_teams and game.